edu-blog-vue/pages/index.vue

```diff
@@ -0,0 +1,170 @@
+<template>
+  <div class="container">
+    <Header
+      image="https://images.unsplash.com/photo-1546074177-ffdda98d214f?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1567&q=80"
+      alt="Header image with letter tiles that says Read More"
+    />
+    <article>
+      <time pubdate :datetime="date" :title="date">{{ date }}</time>
+      <h1>{{ title }}</h1>
+      <p class="lead">
+        {{ lead }}
+      </p>
+      <div v-for="item in content" :key="item.id">
+        <p v-if="item.type === 'paragraph'">
+          {{ item.text }}
+        </p>
+        <ul v-if="item.type === 'ul'">
+          <li v-for="li in item.li" :key="li">{{ li }}</li>
+        </ul>
+        <ol v-if="item.type === 'ol'">
+          <li v-for="li in item.li" :key="li">{{ li }}</li>
+        </ol>
+        <h2 v-if="item.type === 'heading2'">
+          {{ item.text }}
+        </h2>
+        <h3 v-if="item.type === 'heading3'">
+          {{ item.text }}
+        </h3>
+        <h4 v-if="item.type === 'heading4'">
+          {{ item.text }}
+        </h4>
+        <div v-else-if="item.type === 'cols'" class="article-col-container">
+          <div
+            v-for="col in item.col"
+            :key="col"
+            class="article-col-container__col"
+          >
+            <ul v-if="col.type === 'ul'">
+              <li v-for="li in col.li" :key="li">{{ li }}</li>
+            </ul>
+            <ol v-if="col.type === 'ol'">
+              <li v-for="li in col.li" :key="li">{{ li }}</li>
+            </ol>
+          </div>
+        </div>
```

Comment:
Yes! Sorry! My example code with the `div`-wrapping was terrible. You can solve that by taking the `template` tag instead of `div` and change the `:key` to the child. Like this:

```vue
<template v-for="item in content">
  <p v-if="item.type === 'paragraph'" :key="item.id">
    {{ item.text }}
  </p>

…
…
…
</template>
```